cirq-google/cirq_google/experimental/noise_models/calibration_to_fidelity.py

```diff
@@ -0,0 +1,33 @@
+import cirq
+import cirq_google
+import numpy as np
+from cirq.devices.fidelity import Fidelity
+
+def fidelity_from_calibration(calibration: cirq_google.Calibration):
+
+  def unpack_from_calibration(metric_name):
+    if metric_name in calibration.keys():
+      return np.mean([value for qubit, value in calibration[metric_name].items()])
```

Comment:
Add a note about why we take the mean here, and a TODO for supporting per-qubit noise.